jumpscale/core/executors/remote.py

```diff
@@ -35,6 +35,17 @@ def execute(cmd, command_ctx, connection_ctx):
 
 
 class RemoteExecutor:
+    """Executor remote allows executing commands within specific env on the local machine. using the executor framework you can retrieve the stdout, stderr, and the return code as well.
+    e.g
+        with j.core.executors.RemoteExecutor(host="localhost", connect_kwargs={"key_filename":"/home/xmonader/.ssh/id_rsa",}) as c:c.run("hostname")
+
+    Args:
+        connection_ctsx(dict): context passed
```

Comment:
I think what's required in #380 is more explicit info about possible options that can be passed, maybe we can list the most important ones at least?

Reply:
done